plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -237,7 +243,7 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 		return query.page === 'wc-admin' && ! query.path;
 	};
 
-	const isProductPage = () => {
+	const isProductScreen = () => {
```

Comment:
Don't disagree or feel strongly about this change, but just curious if there are reasons to change this name to encompass more screens.

Reply:
>Basically I associate the term screen a bit more with the reactified MVP, and page moreso with the current UI.

I'm fine with either, though I do know we still refer to some areas as "pages."  I guess we should figure out which we like and be consistent, but I'm fine with this change for now.

>I didn't want to use the term "MVP" since that will soon be obsolete

Definitely agree with this.  We need to name what we want it to be going forward and not something "future-y" since it won't always be.